### PR TITLE
[feature](metric) Support collect query counter and error query counter metric in user level

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Issue Number: close #xxx

1.support collect query counter and error query counter metric in user level
2. add sum and count for histogram metric for mistaken delete in PR #22045 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
run buildall

**Comment by caiconghui** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.53 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17166959829 Bytes

**Review by yiguolei — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java [hunk 1]
```diff
@@ -296,10 +296,12 @@ private void auditAfterExec(String origStmt, StatementBase parsedStmt,
 
         if (ctx.getState().isQuery()) {
             MetricRepo.COUNTER_QUERY_ALL.increase(1L);
+            MetricRepo.USER_COUNTER_QUERY_ALL.getOrAdd(ctx.getQualifiedUser()).increase(1L);
             if (ctx.getState().getStateType() == MysqlStateType.ERR
                     && ctx.getState().getErrType() != QueryState.ErrType.ANALYSIS_ERR) {
                 // err query
                 MetricRepo.COUNTER_QUERY_ERR.increase(1L);
+                MetricRepo.USER_COUNTER_QUERY_ALL.getOrAdd(ctx.getQualifiedUser()).increase(1L);
```
Comment (wangbo) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java:304]: Why add USER_COUNTER_QUERY_ALL here?
Isn't it duplicate with row 299?

Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java]: yes，it is a bug

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by caiconghui** [comment]
run buildall

**Review by wangbo — APPROVED** [review]
LGTM

**Review by weizuo93 — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.6 seconds
 stream load tsv:          543 seconds loaded 74807831229 Bytes, about 131 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          28.7 seconds inserted 10000000 Rows, about 348K ops/s
 storage size: 17167150486 Bytes

**Pull request merged**